### PR TITLE
Overlays: Add configurations for accessibility testing

### PR DESCRIPTION
**Pull request by jdvictoria**

### CODE REVIEW

**Comment by jdvictoria** [comment]
[Axe a11y testing: overlays](https://trello.com/c/1rBt6hHa/8836-axe-a11y-testing-overlays)

File: e2e/testcafe-devextreme/tests/accessibility/floatingActionButton.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Properties } from 'devextreme/ui/speed_dial_action.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const options: Options<Properties> = {
+  visible: [true, false],
```
Comment (marker-dao) [inline e2e/testcafe-devextreme/tests/accessibility/floatingActionButton.ts]: Check this comment, please: [https://github.com/DevExpress/DevExtreme/pull/27317#discussion_r1592029176].
Do you agree with it?

Comment (jdvictoria) [reply e2e/testcafe-devextreme/tests/accessibility/floatingActionButton.ts]: I agree.

**Review by marker-dao — DISMISSED** [review]

File: e2e/testcafe-devextreme/tests/accessibility/loadPanel.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Properties } from 'devextreme/ui/load_panel.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const options: Options<Properties> = {
+  showIndicator: [true, false],
```
Comment (EugeniyKiyashko) [inline e2e/testcafe-devextreme/tests/accessibility/loadPanel.ts:11]: Should we declare visible option? By default visible option is false and loadPanel will not rendered after creating

Comment (EugeniyKiyashko) [reply e2e/testcafe-devextreme/tests/accessibility/loadPanel.ts:11]: We should open the LoadPanel and other components after initialization. For this purpose, we may use the show() method or declare corresponding options during initialization to make the panel visible.

File: e2e/testcafe-devextreme/tests/accessibility/tooltip.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const options: Options<Properties> = {
+  visible: [true],
+  target: ['#container'],
```
Comment (EugeniyKiyashko) [inline e2e/testcafe-devextreme/tests/accessibility/tooltip.ts:11]:
```suggestion
  target: [undefined, '#container'],
```

File: e2e/testcafe-devextreme/tests/accessibility/tooltip.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  width: [50],
+  height: [25],
+  disabled: [true, false],
+  hint: [undefined, 'hint'],
+};
```
Comment (EugeniyKiyashko) [inline e2e/testcafe-devextreme/tests/accessibility/tooltip.ts:16]: shading: [true, false],
wrapperAttr: [undefined, { id: "elementId", class: "class-name"]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request merged**